parlai/mturk/tasks/acute_eval/example_script.py

```diff
@@ -3,28 +3,24 @@
 # Copyright (c) Facebook, Inc. and its affiliates.
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
-from parlai.mturk.tasks.acute_eval.run import main as run_main, add_args
+from parlai.mturk.tasks.acute_eval.run import AcuteEvaluator, add_args
 
 """
 Example script for running ACUTE-EVAL.
-The only arguments that *must* be modified for this to be run are:
-'dialogs_path':  Path to folder containing logs of all model conversations.
-    Each model should have one file named '<modelname>.jsonl'
-'model_comparisons': List of tuples indicating pairs of models to be compared.
-    The model names here should match the names of files in the folder
-'onboarding_tasks': List of tuples in format (id1, id2, name of matchup) where
-    id1, id2 are conversation ids where id2 is the correct conversation's id
+The only argument that *must* be modified for this to be run is:
```

Comment:
I wonder if it's worth mentioning args that we MIGHT want to run.

Reply:
i'll add a comment regarding which other args might need tweaking